src/vm-repair/azext_vm_repair/custom.py

```diff
@@ -573,14 +573,15 @@ def reset_nic(cmd, vm_name, resource_group_name, yes=False):
         subnet_id_tokens = subnet_id.split('/')
```

Comment:
Check my previous PR to this file or branch. Version of reset-nic may also needs to be changed.

Reply:
Okay working on it

Reply:
Done